ministryofjustice/analytics-platform-ops
# Add S3 bucket and IAM Role/Policy for user NFS home dir backups

## What

Added:
- S3 bucket with versioning and encryption enabled
- IAM role for backup container
- IAM policy for container access to S3 bucket

## How to review

1. Checkout this branch
2. `terraform plan` and `terraform apply` in dev env
3. Verify that bucket, IAM role and IAM policy are created
4. Install the Helm chart from [this related PR](https://github.com/ministryofjustice/analytics-platform-helm-charts/pull/33) and confirm backups are successful
